docs/modules/ROOT/pages/node_types.adoc

```diff
@@ -120,6 +120,10 @@ The following fields are given when relevant to nodes in the source code:
 
 |forwarded-args|Forwarding arguments into a method call|None|foo(...)|N/A
 
+|forwarded-restarg|Forwarding positional arguments into a method call|None|foo(*)|N/A
```

Comment:
```suggestion
|forwarded-restarg|Forwarding rest arguments into a method call|None|foo(*)|N/A
```

Reply:
Both work; I'm fine with "positional" as it's very clear 👍